tests/azure/storageaccount_test.go

```diff
@@ -0,0 +1,57 @@
+package azure_test
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"testing"
+
+	"github.com/NeowayLabs/klb/tests/lib/azure"
+	"github.com/NeowayLabs/klb/tests/lib/azure/fixture"
+	"github.com/NeowayLabs/klb/tests/lib/nash"
+)
+
+func genStorageAccountName() string {
+	return fmt.Sprintf("klbstortests%d", rand.Intn(1000))
+}
+
+func testStorageAccountCreate(t *testing.T, f fixture.F) {
+	genstorage := genStorageAccountName()
+	nash.Run(
+		f.Ctx,
+		t,
+		"./testdata/create_storage_account.sh",
+		f.ResGroupName,
+		genstorage,
+		f.Location,
+	)
+	storage := os.Getenv("STORAGE_ACCOUNT_NAME")
```

Comment:
@tiago4orion I'm very interested on alternatives to communicate Go <-> Nash script but right now I'm going to merge this, and we can improve it later